components/tabcordion/examples/50-local-tokens.js

```diff
@@ -0,0 +1,116 @@
+/** @jsx jsx */
+
+import { forwardRef } from 'react';
+import { GEL, jsx } from '@westpac/core';
+import { Tab, Tabcordion } from '@westpac/tabcordion';
+import { data } from './_data';
+
+const Panel = forwardRef(({ isLast, isSelected, ...props }, ref) => {
+	const common = {
```

Comment:
Not to be silly but can't we just put the thing in there? What's the point of the `common` var here?

Reply:
Yea sorry I forgot to move it into the element itself. I find it easier to move out the styles when I'm working on a component and then move them back when im done.